appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt

```diff
@@ -114,17 +114,19 @@ internal sealed class ExperienceLifecycleEvent(
         this["version"] = experience.publishedAt
         this["localeName"] = experience.localeName
         this["localeId"] = experience.localeId
+        this["workflowId"] = experience.workflowId?.appcuesFormatted()
+        this["workflowTaskId"] = experience.workflowTaskId?.appcuesFormatted()
         this["trigger"] = triggerValue
 
         when (val trigger = experience.trigger) {
             is ExperienceCompletionAction -> {
-                this["fromExperienceId"] = trigger.fromExperienceId
+                this["fromExperienceId"] = trigger.fromExperienceId.appcuesFormatted()
             }
             is LaunchExperienceAction -> {
-                this["fromExperienceId"] = trigger.fromExperienceId
+                this["fromExperienceId"] = trigger.fromExperienceId?.appcuesFormatted()
             }
             is PushNotification -> {
-                this["pushNotificationId"] = trigger.fromPushId
+                this["pushNotificationId"] = trigger.fromPushId.appcuesFormatted()
```

Comment:
I think these are relatively new (in 4.0 here https://github.com/appcues/appcues-android-sdk/commit/f9914a8936a9298de1aaf8c68be96e3134be8131), but noticed they were not formatted UUID values like the rest, so updating here. I don't think anything is actually using this data, yet.